deepfence_worker/cronscheduler/init_db.go

```diff
@@ -73,12 +73,12 @@ func initSqlDatabase(ctx context.Context) error {
 	return nil
 }
 
-func InitMinioDatabase() error {
```

Comment:
Do we want to continue if MinioDatabase init fails?
I think we should exit early if it fails

Reply:
will panic on error so that scheduler crashes and we can detect error